src/extension/background-script/actions/lnurl/pay.ts

```diff
@@ -0,0 +1,20 @@
+import utils from "../../../../common/lib/utils";
+
+import type { Message, LNURLDetails } from "../../../../types";
+
+async function payWithPrompt(message: Message, lnurlDetails: LNURLDetails) {
+  // for lnurl pay requests we always open a prompt
+  // Question: does it help to support an allowance here?
+  try {
+    const response = await utils.openPrompt({
+      origin: message.origin,
+      type: "lnurlPay",
+      args: { ...message.args, lnurlDetails },
+    });
+    return response; // { data: ... }
```

Comment:
What's this comment?

Reply:
was just for me to say what's in response. but TS says exactly that.